lib/taskana-core/src/main/java/pro/taskana/common/internal/jobs/AbstractTaskanaJob.java

```diff
@@ -42,6 +45,14 @@ public static TaskanaJob createFromScheduledJob(
         return new TaskCleanupJob(engine, txProvider, job);
       case WORKBASKETCLEANUPJOB:
         return new WorkbasketCleanupJob(engine, txProvider, job);
+      case HISTORYCLEANUPJOB:
```

Comment:
Didn't we agree to change this mechanism to reflection to prevent this kind of hard dependency?